### PR TITLE
fix flutter install command : pub get -> pub add

### PR DESCRIPTION
**Pull request by imtushardave**
## What does this PR do?

Fix flutter installation command - pub get to pub add

## Related PRs and Issues

([5277](https://github.com/appwrite/appwrite/issues/5277))
### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Review by DHsoftw — APPROVED** [review]
Seems correct

File: app/views/docs/getting-started-for-flutter.phtml [hunk 1]
```diff
@@ -121,7 +121,7 @@ close the window.
 <p>You can also install the SDK using the <a href="https://pub.dev/" target="_blank" rel="noopener">Dart package manager</a> from your terminal:</p>
 
 <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
-  <pre class="line-numbers"><code class="prism language-bash" data-prism>pub get appwrite</code></pre>
+  <pre class="line-numbers"><code class="prism language-bash" data-prism>pub add appwrite</code></pre>
```
Comment (stnguyen90) [inline app/views/docs/getting-started-for-flutter.phtml:124]: This won't actually run. The correct command should be:

```suggestion
  <pre class="line-numbers"><code class="prism language-bash" data-prism>flutter pub add appwrite</code></pre>
```

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

File: app/views/docs/getting-started-for-flutter.phtml (hunk 1 above)
Comment (imtushardave) [reply app/views/docs/getting-started-for-flutter.phtml]: Fixed

**Review by stnguyen90 — APPROVED** [review]

**Review by abnegate — APPROVED** [review]

**Pull request merged**